src/SubscribeMe/Subscriber/MailchimpSubscriber.php

```diff
@@ -113,13 +119,82 @@ public function subscribe(string $email, array $options, array $userConsents = [
                      */
                     return true;
                 }
+                if ($body['id'] !== null) {
+                    return $body['id'];
+                }
+                return false;
             }
-
-            throw new CannotSubscribeException($exception->getMessage(), $exception);
-        } catch (RequestException $exception) {
+        } catch (ClientExceptionInterface $exception) {
             throw new CannotSubscribeException($exception->getMessage(), $exception);
         }
 
         return false;
     }
+
+    /**
+     * @see https://mailchimp.com/developer/transactional/api/messages/send-using-message-template/
+     * @param array<EmailAddress> $emails
+     * @param array $variables
+     * @param string $templateEmail
+     * @return string
+     */
+    public function sendTransactionalEmail(array $emails, array $variables, string $templateEmail): string
+    {
+        if (empty($emails)) {
+            throw new \InvalidArgumentException('Emails information missing');
+        }
+
+        $recipients = array_map(function (EmailAddress $emailAddress) {
+            return [
+                'email' => $emailAddress->getEmail(),
+                'name' => $emailAddress->getName(),
+                'type' => 'to'
+            ];
+        }, $emails);
+
+        if (empty($variables)) {
+            throw new \InvalidArgumentException('Variables missing');
+        }
+
+        if (empty($templateEmail)) {
+            throw new \InvalidArgumentException('Template Id missing');
+        }
+
+        if (!is_string($this->getApiKey())) {
+            throw new \InvalidArgumentException('ApiKey is not a string');
+        }
+
+        try {
+            $body = [
+                'template_name' => $templateEmail,
+                'template_content' => [
+                    'name' => $templateEmail,
+                    'content' => $variables,
+                ],
+                'message' => [
+                    'to' => $recipients,
+                ],
+                'key' => $this->getApiKey(),
+            ];
+
+            if (!is_string(json_encode($body))) {
+                throw new \InvalidArgumentException('Body missing');
+            }
+            $body = $this->getStreamFactory()->createStream(json_encode($body));
+
+            $url = 'https://mandrillapp.com/api/1.0/messages/send-template';
+
+            $request = $this->getRequestFactory()
+                ->createRequest('POST', $url)
+                ->withBody($body)
+                ->withAddedHeader('Content-Type', 'application/json')
+                ->withAddedHeader('User-Agent', 'rezozero/subscribeme');
+
+            $response = $this->getClient()->sendRequest($request);
+
+            return $response->getBody()->getContents();
```

Comment:
Unnecessary extracted variable `$response`

```suggestion
            return $this->getClient()->sendRequest($request)->getBody()->getContents();
```